.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "_resources/extensions/SubpageWatchlist"]
 	path = _resources/extensions/SubpageWatchlist
 	url = https://gerrit.wikimedia.org/r/mediawiki/extensions/SubpageWatchlist.git
-	branch = REL1_35
+	branch = REL1_39
```

Comment:
I think we should remove the branch completely, as the only scenario where this takes effect is when the `--remote` flag is supplied to the `git submodule update` command. The default behavior for submodules is to checkout the commit that's fixed by the parent repo, and that's how we use submodules